### DISCUSSION TITLE
Person left, but remains detected

### DISCUSSION THREAD

**Discussion by kenni** [blakeblackshear/frigate#8040] [Ask A Question]
Sorry for this basic question, but I can't seem to find an answer in the documentation.

A person gets correctly detected by Frigate and an event is initiated as expected. The person leaves the driveway, but the event doesn't stop. It has now been recording for 3 hours. According to the Bounding Box, the person is still registered as being present. This is of course not true, but how to I avoid this? The size of the Bounding Box is realistic in terms of a size of a person, so I can't fix it by setting a minimum size. Also, I do not want to completely remove that area, as I want it to trigger an event if a person is in fact there.

I've checked the Bounding Box a couple of times over the past hour and it changes its size, although it remains in the same area.

What is the best way to fix this? Thanks in advance.

![image](https://github.com/blakeblackshear/frigate/assets/486971/a8fdc9be-1f91-41a0-959e-d9a81c651929)

**Comment by blakeblackshear** [comment]
Typically this happens when there is a motion mask configured to block off areas like this which is why the documentation specifically recommends not using motion masks in this way. You should be using a required zone to limit events to the porch, not masking off everything else.

**Reply by kenni** [reply]
Ok, thanks a lot for the clarification. Are you referring to the quoted Reddit question in the Masks-section of the documentation? I did read that prior to posting, but the user in question had used Motion Masks to cover everything except his stoop, to which I believe you(?) recommend him to use a required_zone instead. I only have one small squared Motion Mask covering the road, nothing else is masked - very similar to the Motion Mask example depicted on the same page in the documentation.

Do I understand you correctly that we should not use Motion Masks anywhere where the detected object might move to? Eg. only use it for treetops, etc? If that is the case, then the given example in the documentation is a bit misleading, as that is exactly how I have configured my motion mask. It would be nice to have this info at the very beginning of the Motion Mask description to avoid confusion and point users to the required_zone option instead.

I'll attempt with a required zone, thanks, much appreciated :)
